git.txt

```diff
@@ -13,3 +13,11 @@ how to copy local repo to remote repo
 9.git push -set-upstream origin jeff1
 	that means you will create new branch in remote end 
 	then you can push request in remote end.
+
+
+2) git abandon local code force update 
+git fetch --all
+git reset --hard origin/master
+
+3) about git sample flow
+http://blog.jobbole.com/53573/
```

Comment:
maybe the steps of how to create a repo is not correct .next time ,i will test it and modify it.